### PR TITLE
Payment card icons, checkbox/radio fix

### PR DESCRIPTION
**Pull request by oliverviljamaa**
- `checkbox-lg` and `radio-lg` checkbox/radio position fix
- payment card icons added as components
- `transform-style` mixin

### CODE REVIEW

File: less/checkbox-radio.less [hunk 1]
```diff
@@ -139,8 +139,8 @@
         display: block;
     }
 
-    &.checkbox-lg,
-    &.radio-lg {
+    .checkbox-lg,
+    .radio-lg {
```
Comment (StevePole) [inline less/checkbox-radio.less:143]: Not sure if this will break other stuff...?

**Comment by StevePole** [comment]
Not sure we should use 'icons' as the name of the less file and docs section, there will be a new set of icons from design. Perhaps this should be more specific.  Also in most places the title of the docs section is the same as the class used in HTML e.g. alert, jumbotron etc.  

How large is the icons.less file, somewhat concerned about number of inline SVGs and impact on file size.

File: less/checkbox-radio.less (hunk 1 above)
Comment (oliverviljamaa) [inline less/checkbox-radio.less:143]: It was wrong before, `.form-group` shouldn't have these classes, they should be in their child `div`. I'll go over it again and see whether it makes sense to use `>` or put it somewhere else. As we're selecting only `.checkbox-lg` and `.radio-lg` in `.form-group`, it shouldn't leak though.

**Comment by oliverviljamaa** [comment]
@StevePole I thought about that, maybe 'SVG icons'? And good point, it's 45kB, so pretty big. Do you think not inlining them is better?

**Comment by StevePole** [comment]
I think there's a way to use an inline svg, but have the svg reference a URL rather than embed the data, which might be a good solution.  I haven't tried it though, and we need to think about relative vs absolute paths.

**Pull request closed without merging**